### DISCUSSION TITLE
type 类型影响 JSON Schema 联动问题

### DISCUSSION THREAD

**Discussion by chenWebCoder** [alibaba/formily#2972] [Q&A 2.x] [answered]
如例所示：将 Schema 配置中 CustomForm 的 type 由 void 设置为 object 后，原本配置的联动将会失效！请问是是什么原因？

Demo: https://codesandbox.io/s/epic-hypatia-4udyts?file=/App.tsx

**Comment by janryWang** [comment, marked as answer]
多看文档，先理解什么是void，再理解什么是路径系统
